edk2toolext/environment/version_aggregator.py

```diff
@@ -46,6 +46,12 @@ def ReportVersion(self, key, value, versionType):
         }
         self._logger.debug("version_aggregator logging version: {0}".format(str(self.Versions[key])))
 
+    def Print(self):
+        """ Prints out the current information from the version aggregator """
+        for version_key in self.Versions:
+            version = self.Version[version_key]
+            print(f"{version['type']} - {version['name']}: {version['version']}")
```

Comment:
Are these objects? should we instead provide a str function so that every place this is desired don't have to write the same format string?

Reply:
They're dictionaries :/